runtime/etc/mli/mli_server_runtime.c

```diff
@@ -47,6 +47,7 @@ void chpl_mli_server_init(struct chpl_mli_context* server) {
   if (server->context) { return; }
 
   server->context = zmq_ctx_new();
+  server->setup_sock = zmq_socket(server->context, ZMQ_PUSH);
```

Comment:
What's the specific reason behind this being ZMQ_PUSH? Does it matter? What is the lifetime of this socket? (Just curious.)

Reply:
In this case, I'm expecting to just grab as many ports as are needed.  I want to say there's some assurance that we'll always here whatever is passed, so there's isn't as much point doing ACKs ourselves, but I'd have to double check to be certain.

My expectation is that this would not be necessary after all the other sockets have been established, so we could probably disconnect it then.